test/conftest.py

```diff
@@ -0,0 +1,45 @@
+"""test/conftest.py: dynamic testing configuration for pytest
+
+See the pytest documentation for more details:
+https://docs.pytest.org/en/latest/contents.html
+"""
+
+import sys
+
+
+version_major = sys.version_info.major
+version_minor = sys.version_info.minor
+
+# Paths that should be ignored for all Python versions.
+paths_ignore_allver = [
+    'src/cclib/progress/qt4progress.py',
+]
+
+# Paths that should run only for Python 2.7.
+paths_ignore_only_2_7 = [
+    'src/cclib/bridge/cclib2pyquante.py',
+]
+
+# Paths that should run everywhere except Python 3.2.
+paths_ignore_not_3_2 = [
+    'src/cclib/bridge/cclib2biopython.py',
+]
+
+
+def match_path(path, partial_paths):
+    """Does the given path contain any of the stubs in partial_paths?"""
+    return any(partial_path in str(path)
+               for partial_path in partial_paths)
+
+
+def pytest_ignore_collect(path, config):
```

Comment:
I don't like this for one reason: we're moving the controls for exceptions away from the tests. How are we supposed to keep track of this, presuming we'd eventually like to get rid of these exceptions?